NEWS.md

```diff
@@ -2,6 +2,8 @@
 
 ## Development
 
+* Do not render README.qmd to markdown automatically. Users should render them manually to make sure that the README on CRAN and Github is in sync with the Altdoc home page.
```

Comment:
can you put this in a "Breaking changes" category?

Reply:
https://github.com/etiennebacher/altdoc/pull/302/commits/8ead69020f0b53d6102ad5202e9d8efe61468f6b